.gitignore

```diff
@@ -1,23 +1,6 @@
-# Compiled class file
-*.class
+target/
```

Comment:
Could you please add this for VS Code users :P 
```
# VSCode
/.vscode
/.settings
.classpath
.factorypath
.project
```

Reply:
done